pridepy/files/files.py

```
@@ -58,11 +59,156 @@ class Files:
+
+        return checksums
+
+    @staticmethod
+    def compute_md5(file_path: str, chunk_size: int = 4 * 1024 * 1024) -> str:
+        """
+        Compute an MD5 checksum for integrity validation, not for security use.
+        """
+        try:
+            md5 = hashlib.md5(usedforsecurity=False)
+        except TypeError:
+            md5 = hashlib.md5()
```

Ruff (0.15.10)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)